capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraActivity.java

```diff
@@ -390,12 +396,29 @@ private void restoreSavedState(@Nullable final Bundle savedInstanceState) {
     }
 
     private void setTitleOnTablets() {
+        if (getSupportActionBar() != null) {
+            getSupportActionBar().setTitle(isTablet() ? getString(R.string.gc_camera_title) : getString(R.string.gc_title_camera));
+        }
+    }
 
-        boolean isTablet = getResources().getBoolean(R.bool.gc_is_tablet);
+    private void setupCameraBottomNavigationBar() {
+        if (GiniCapture.hasInstance() && GiniCapture.getInstance().isBottomNavigationBarEnabled()) {
+            InjectedViewContainer<CameraBottomNavigationBar> injectedViewContainer =
+                    findViewById(R.id.gc_injected_navigation_bar_container_bottom);
+            CameraBottomNavigationBarAdapter adapter = GiniCapture.getInstance().getCameraBottomNavigationBarAdapter();
+            injectedViewContainer.setInjectedViewAdapter(adapter);
 
-        if (getSupportActionBar() != null) {
-            getSupportActionBar().setTitle(isTablet ? getString(R.string.gc_camera_title) : getString(R.string.gc_title_camera));
+            adapter.setTitle(isTablet() ? getString(R.string.gc_camera_title) : null);
+
+            adapter.setOnBackButtonClickListener(v -> onBackPressed());
+
+            adapter.setOnHelpButtonClickListener(v -> startHelpActivity());
         }
+
+    }
+
+    private boolean isTablet() {
```

Comment:
[Suggestion] We have a helper method for this called `ContextHelper.isTablet()`. Sorry for not noticing this earler.

Reply:
Done